### PR TITLE
feat(levm): add `levm` crate

### PR DESCRIPTION
**Pull request by ilitteri**
**Description**

Add Lambda's EVM implementation (LEVM).

- Adds `levm` crate implementation.
- Adds `levm` crate and benches to the workspace.
- Adds workflow for running `levm`'s benches.

### CODE REVIEW

File: .github/workflows/levm_bench.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: LEVM benchmarks
```
Comment (mpaulucci) [inline .github/workflows/levm_bench.yaml:1]: you also want https://github.com/lambdaclass/lambda_ethereum_rust/commit/0a93d977a890ae4404f80bc3f70636073cda3dfc#diff-fbedb504135d1fa140a0a84c8f428de49626a6b7a2ee68a17da5d941474b22a8

Comment (ilitteri) [reply .github/workflows/levm_bench.yaml:1]: [57ea30a](https://github.com/lambdaclass/lambda_ethereum_rust/pull/869/commits/57ea30a5b2f410f5640f9c6227ce9d8273f14e8d)

Comment (jrchatruc) [reply .github/workflows/levm_bench.yaml:1]: EF tests will be added later so we don't want that workflow right now

**Review by mpaulucci — APPROVED** [review]
LGTM

**Review by jrchatruc — APPROVED** [review]

**Pull request merged**